app.js

```diff
@@ -1,9 +1,15 @@
 const express = require("express");
 const app = express();
-const { getAllTopics, endpointsDescription } = require("./controllers/controller");
+const {
+  getAllTopics,
+  endpointsDescription,
+  sendArticle,
+} = require("./controllers/controller");
 
 app.get("/api", endpointsDescription);
 
 app.get("/api/topics", getAllTopics);
 
+app.get("/api/articles/:id", sendArticle);
+
```

Comment:
at this point we can introduce our error handling middleware chain - handle PSQL errors, handle custom errors and handle server errors